common/docker-entrypoint.sh

```diff
@@ -32,16 +32,11 @@ parseBoolean() {
 
 # This line is an addition to the NGINX Docker image's entrypoint script.
 if [ -z ${DNS_RESOLVERS+x} ]; then
-  resolvers=""
-  for ip in $(grep nameserver /etc/resolv.conf | cut -d' ' -f2 | xargs)
-  do
-    if echo "${ip}" | grep -q ':'; then
-      resolvers="$resolvers [${ip}]"
-    else
-      resolvers="$resolvers $ip"
-    fi
-  done
-  export DNS_RESOLVERS="${resolvers}"
+   # This method of pulling individual nameservers from
+   # /etc/resolv.conf taken from the entrypoint script in the
+   # official docker image.
+   # https://github.com/nginxinc/docker-nginx/blob/master/entrypoint/15-local-resolvers.envsh
+  export DNS_RESOLVERS="$(awk 'BEGIN{ORS=" "} $1=="nameserver" {print $2}' /etc/resolv.conf)"
```

Comment:
Does this awk pattern support ipv6 syntax? In order to template in ipv6 ips into nginx config, they must be enclosed in `[]`. This was raised in [this bug](https://github.com/nginxinc/nginx-s3-gateway/issues/83).
The fix was [here](https://github.com/nginxinc/nginx-s3-gateway/commit/97a87377bdda5de8d61442d81514cd61ed48ee32).

Reply:
Thank you for catching this! I had overlooked this case and missed the enclosure in square brackets from the original code.  I've made changes in [this commit](https://github.com/nginxinc/nginx-s3-gateway/pull/176/commits/0c1d73234c387f2a47f4e2150da3ae7595b3213e) that combines your fix with the changes from the official dockerfile entrypoint.

Given a `resolv.conf` that looks like this:
```
nameserver 127.0.0.11
nameserver 8.8.4.4
nameserver 94.198.184.14
nameserver 94.198.184.34
# NOTE: the libc resolver may not support more than 3 nameservers.
# The nameservers listed below may not be recognized.
nameserver 8.8.8.8
# This is another comment
nameserver 8.8.8.9
nameserver fe80::f03c:93ff:fe25:6762
nameserver 8.9.9.9
nameserver 2a03:a800:2:1::1
nameserver 8.9.9.7
options ndots:0
```

We see:
```
DNS Resolvers:  127.0.0.11 8.8.4.4 94.198.184.14 94.198.184.34 8.8.8.8 8.8.8.9 [fe80::f03c:93ff:fe25:6762] 8.9.9.9 [2a03:a800:2:1::1] 8.9.9.7
```

Which looks right to me. Based on my read of the `resolver` syntax each address should be separately enclosed in square brackets.